src/views/Order.vue

```diff
@@ -1,118 +1,279 @@
+                <ion-select-option value="any">any</ion-select-option>
+              </ion-select>
+            </ion-item>
+          </ion-list>
+          <ion-list>
+            <ion-list-header>Type</ion-list-header>
+            <ion-item>
+              <ion-label>order created</ion-label>
+              <ion-checkbox/>
+            </ion-item>
+            <ion-item>
+              <ion-label>order created</ion-label>
+              <ion-checkbox/>
+            </ion-item>
+            <ion-item>
+              <ion-label>order created</ion-label>
+              <ion-checkbox/>
+            </ion-item>
+          </ion-list>
+          <ion-list>
+            <ion-list-header>Date</ion-list-header>
+            <ion-item>
+              <ion-label>order created</ion-label>
+              <ion-select value="any">
+                <ion-select-option value="any">any</ion-select-option>
+              </ion-select>
+            </ion-item>
+            <ion-item>
+              <ion-label>order created</ion-label>
+              <ion-select value="any">
+                <ion-select-option value="any">any</ion-select-option>
+              </ion-select>
+            </ion-item>
+            <ion-item>
+              <ion-label>order created</ion-label>
+              <ion-select value="any">
+                <ion-select-option value="any">any</ion-select-option>
+              </ion-select>
+            </ion-item>
+          </ion-list>
+          <ion-card>
+            <ion-toolbar>
+              <ion-title>Purchase date</ion-title>
+            </ion-toolbar>
 
+            <ion-card-content>
+              <ion-chip>
+                <ion-label>PO #</ion-label>
+              </ion-chip>
+              <ion-chip>
+                <ion-label>PO #</ion-label>
+              </ion-chip>
+            </ion-card-content>
+          </ion-card>
         </div>
-        <div class="order-metatags">
-          <ion-note>Ordered on 7 Jan 2021</ion-note>
-          <ion-badge color="primary">Approved</ion-badge>
-        </div>
+        <div class="order-detail">
+          <div class="order-header">
+            <div class="order-id">
+              <ion-item lines="none">
+                <ion-label>
+                  <h1>Order ID</h1>
+                  <p>Customer Name</p>
+                </ion-label>
+              </ion-item>
+            </div>
+            <div class="order-tags">
+              <ion-chip>
+                <ion-icon :icon="pricetag" />
+                <ion-label>Shopify ID</ion-label>
+              </ion-chip>
+              <ion-chip>
+                <ion-icon :icon="pricetag" />
+                <ion-label>Customer Loyalty Status</ion-label>
+              </ion-chip>
+            </div>
 
+            <div class="order-metatags">
+              <ion-note class="metatags">Ordered on 7 Jan 2021</ion-note>
+              <div class="tags">
+                <ion-badge color="primary" slot="end">Approved</ion-badge>
```

Comment:
Why primary color value is passed over here ?

Reply:
done